### ISSUE TITLE
feat(docs): add BlitzH documentation [2h]

### ISSUE THREAD

**Issue by mesqueeb** [mesqueeb/blitzar#17]
I want the blitzar schema concept to be usable as a replacement for render functions: https://v3.vuejs.org/guide/render-function.html#v-on

The schema concept is really user friendly and easy to use.

But we need a more lightweight option without it necessarily wrapping a `<form>` and without the concept of "labels", "label position" etc.

PS: the `h()` of render functions stands for [HyperScript](https://css-tricks.com/what-does-the-h-stand-for-in-vues-render-method/)

initial implementation available at https://github.com/cycraft/blitzar/blob/production/packages/form/src/components/BlitzH.vue

- [ ] I really need to flesh out this idea
- [ ] write some basic documentation
- [ ] determine other tasks

**Comment by eyleron** [comment]
So is the concept of BlitzH that it'd be used if one wanted to use schema to render non-form components? Lists, cards, panels, maps, etc.?

**Comment by mesqueeb** [comment]
@eyleron yes! Still not sure how verbal I want to be about this component. But it's already used throughout the library to render div fields etc.
